### ISSUE TITLE
JS Not working in PW3.0.42

### ISSUE THREAD

**Issue by somatonic** [MadeMyDay/PageTableExtended#23]
This check added
//  https://processwire.com/talk/topic/5477-check-if-a-module-is-loaded/
		if(!class_exists("InputfieldPageTable", false)) {
			$this->config->scripts->add($this->config->urls->InputfieldPageTable . "InputfieldPageTable.js");
			$this->config->styles->add($this->config->urls->InputfieldPageTable . "InputfieldPageTable.css");
		}

Doesn't seem to work and the js doesn't get added. Thus it doesn't work on the js side.

**Comment by MadeMyDay** [comment]
Right. Bummer. Do you know a method to check if a certain module is already loaded?

**Comment by somatonic** [comment]
I don't understand why this check is needed anyway?

**Comment by MadeMyDay** [comment]
Well, #19

**Comment by somatonic** [comment]
Sorry just noticed it was on 2.73 not 3.0.42!!

Anyway I tested the case with PageTable and PTE field on same page and it's working without this check (like in older version). So I'm not sure what the problem is or was. I can't reproduce it.

**Comment by MadeMyDay** [comment]
Reverted back for now. Thanks Soma!
